File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    // Get dollars from user
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);


    // Unit conversion
    int cents = round(dollars * 100);


    // Calculate number of coins
    int coins = 0;
    while (cents > 24)
    {
        cents = cents - 25;
        coins++;
    }

    while (cents > 9)
    {
        cents = cents - 10;
        coins++;
    }

    while (cents > 4)
    {
        cents = cents - 5;
        coins++;
    }

    while (cents > 0)
    {
        cents = cents - 1;
        coins++;
    }


    // Print number of coins
    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

count = 0

# Get dollars from user
while True:
    dollars = get_float("Change owed: ")
    if dollars > 0:
        break

# Unit conversion
cent = round(dollars * 100)

# Caluculate number of count
while cent > 24:
    cent = cent - 25
    count += 1

while cent > 9:
    cent = cent - 10
    count += 1

while cent > 4:
    cent = cent - 5
    count += 1

while cent > 0:
    cent = cent - 1
    count += 1

# Print number of count
print(count)